include/miopen/miopen.h

```diff
@@ -339,7 +345,9 @@ typedef enum
         4, /*!< Pack of four 8-bit int points in NCHW_VECT_C format (Partially supported) */
     miopenBFloat16 = 5, /*!< 16-bit binary floating point (8-bit exponent, 7-bit fraction)
                            (Partially supported) */
-    miopenDouble = 6,   /*!< 64-bit floating point (Partially supported) */
+    miopenDouble  = 6,  /*!< 64-bit floating point (Partially supported) */
+    miopenFloat8  = 7,
+    miopenBFloat8 = 8
```

Comment:
[Recommendation] Please add documentation - explain the difference between FP8 and BF8.

Reply:
@xinlipn Thanks! What I am thinking about is adding this to the doxygen doc. Does it sound reasonable for you?